doc/man3/EVP_EncryptInit.pod

```diff
@@ -156,6 +160,16 @@ EVP_CIPHER_CTX_mode
  int EVP_Cipher(EVP_CIPHER_CTX *ctx, unsigned char *out,
                 const unsigned char *in, unsigned int inl);
 
+ int EVP_CIPHER_can_pipeline(const EVP_CIPHER *cipher, int enc);
+ int EVP_CipherPipelineInit(EVP_CIPHER_CTX *ctx, const EVP_CIPHER *cipher,
+                                  const unsigned char *key, size_t numpipes,
+                                  const unsigned char **iv, int enc);
+ int EVP_CipherPipelineUpdate(EVP_CIPHER_CTX *ctx,
+                                    unsigned char **out, size_t *outl,
+                                    const unsigned char **in, const size_t *inl);
+ int EVP_CipherPipelineFinal(EVP_CIPHER_CTX *ctx,
+                                   unsigned char **outm, size_t *outl);
```

Comment:
I would add the explicit outsize input arguments here as well. There is no reason to copy the deficiencies of the non-pipelining functions here.

Reply:
Do we need the same for `EVP_CipherPipelineUpdate()` or assume `out` buffer to be of same size as `in` buffer?

Reply:
Done